examples/1_standalone_app/README.md

```diff
@@ -0,0 +1,103 @@
+# Standalone App Snippet Example
+
+This tutorial shows you how to create a standalone Mobly snippet app, which runs
+by itself and doesn't instrument a main app.
+
+## Tutorial
+
+1.  Use Android Studio to create a new app project.
+
+1.  Link against Mobly Snippet Lib in your `build.gradle` file
```

Comment:
That's on purpose; markdown auto-numbers lists regardless of what the numbers in the .md are. I could have made them all 20 if I wanted and would still get a nice numbered list.

Having them all say 1. is preferred for longer lists so you don't have to manually renumber them when you add points in the middle or shuffle steps around.